tests/snapshot/snapshot-demos-form-QueryFilter-light-filter.html

```
@@ -933,11 +909,49 @@
+                            class="ant-pro-core-field-label ant-pro-core-field-label-middle ant-pro-core-field-label-bordered"
+                          >
+                            区域
                             <span
-                              style="display: inline-flex; align-items: center;"
+                              aria-label="down"
+                              class="anticon anticon-down ant-pro-core-field-label-icon ant-pro-core-field-label-arrow"
+                              role="img"
                             >
-                              <span
-                                class="ant-pro-core-field-label-text"
+                              <svg
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </svg> ], start tag match failed [ <svg
                                aria-hidden="true"
                                data-icon="down"
                                fill="currentColor"
                                focusable="false"
                                height="1em"
                                viewBox="64 64 896 896"
                                width="1em"
                              > ] on line 947.

(tag-pair)